docs/source/reference/supported_apis.rst

```diff
@@ -0,0 +1,1207 @@
+.. _implementation/dataframe_supported:
+
+=====================
+Supported Pandas APIs
+=====================
+
+The following table is structured as follows: The first column contains the method name.
+The second column is a flag for whether or not Opensearch-py-ml supports the method on the
+corresponding object.
+
+ .. note::
+
+    Even if an interface is listed here as "supported"
+    doesn't mean all parameters are implemented. Sometimes only a subset of parameters
+    can be supported.
+
+If you have need of an operation that is listed as not implemented,
+feel free to `open an issue <https://github.com/opensearch-project/opensearch-py-ml/issues>`_
+or give a thumbs up to already created issues. Contributions are also welcome!
+
+ .. note::
+
+    Some Pandas methods are not implementable due to the constraints of the
+    library using Opensearch as the backend for the majority of data
+    processing. Functions like ``DataFrame.iloc[i]`` or ``DataFrame.transpose()``
+    likely won't be implementable in Opensearch-py-ml due to this constraint.
+
+This list should be automatically generated with ``utils/generate-supported-apis.py``
+script instead of being modified manually.
+
++---------------------------------------+------------+
+| Method or Property                    | Supported? |
++=======================================+============+
+| ``ed.DataFrame.abs()``                | No         |
```

Comment:
What does `ed` mean?

Reply:
Good Question. `ed` is used as short form of `eland`. I was looking at that file. This file is auto generated. I pushed a commit to delete this file and later I will fork the code to generate this rst by itself.

Reply:
Cool, actually I see `ed` in other files too.

Reply:
Yeah, those are in the code files. To reflect/change the variable name, requires another larger PR, which I'm planning to do after this PR.